tests/conftest.py

```diff
@@ -21,8 +22,27 @@
 warnings.filterwarnings('ignore',
                         message='Signal.put no longer takes keyword arguments')
 # Other temporary patches to FakeEpicsSignal
+def check_value(self, value):
```

Comment:
Should upstream this and others. Looks good though.

Reply:
Yeah, I kind of want to upstream this and I kind of want to avoid it because I'll fall down the "make it actually good" rabbit hole...

Reply:
Fair!